evaluation/logic_reasoning/.cache_program/facts.kfb

```diff
@@ -0,0 +1,12 @@
+Cold(Bob, True)
```

Comment:
Maybe we can also remove the `.cache_program` directory?

Reply:
It seems the directory get used in `evaluation/logic_reasoning/logic_inference.py`

Reply:
yes, this ```.cache_program``` directory is necessary.